osu.Game/Database/IPostImports.cs

```diff
@@ -12,6 +12,6 @@ public interface IPostImports<TModel>
         /// <summary>
         /// Fired when the user requests to view the resulting import.
         /// </summary>
-        public Action<IEnumerable<Live<TModel>>>? PostImport { set; }
+        public Action<IEnumerable<Live<TModel>>>? PresentImport { set; }
```

Comment:
The interface this is on - `IPostImports` - now has a weird name and is generally pretty useless (nothing ever actually uses it). Probably not worth fixing, should be merged into something else as a follow-up if anything.

Reply:
Good catch, I've removed the interface and moved the method to `IModelImporter`. Still seems sane to have it there as long as `BeatmapManager` is doing the delgation.